migrations/notifications.py

```diff
@@ -6,7 +6,7 @@
 Create `notifications` table.
 """
 
-from gdcdatamodel.models.notifications import Base, Notification
+from gen3datamodel.models.notifications import Base, Notification
```

Comment:
i still see some references to "gdcdatamodel" in the repo (docstrings, logger name, and docs/examples directories) - should we do a global search and replace everywhere?

Reply:
Yes, I think I finally got all those.